### PR TITLE
dm-tool: add page

### PR DESCRIPTION
**Pull request by godalming123**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

docs on tool: https://www.systutorials.com/docs/linux/man/1-dm-tool/

### CODE REVIEW

File: pages/linux/dm-tool.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# dm-tool
```
Comment (einverne) [inline pages/linux/dm-tool.md:1]: Need an empty line.

File: pages/linux/dm-tool.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+A tool to commnicate with the lightdm display manager. EG: lock currect session and show lightdm, list all open sessions, connect an existing x server to the display manager.
```
Comment (einverne) [inline pages/linux/dm-tool.md:2]:
```suggestion
> A tool to control the display manager.
```

File: pages/linux/dm-tool.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+More informatioin: https://www.mankier.com/1/dm-tool.
```
Comment (einverne) [inline pages/linux/dm-tool.md:3]:
```suggestion
> More information: <https://www.mankier.com/1/dm-tool>.
```

File: pages/linux/dm-tool.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+- show the greeter while keeping current desktop session in ram and waiting to be restored upon autentication by logged in uuser
+`dm-tool switch-to-greeter`
```
Comment (EmilySeville7cfg) [inline pages/linux/dm-tool.md:6]:
```suggestion
- Show the greeter while keeping current desktop session in ram and waiting to be restored upon authentication by logged in user:

`dm-tool switch-to-greeter`
```

File: pages/linux/dm-tool.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+- swicth user, showing a authentication prompt if required
+`dm-tool swict-to-user {{username}} [{{session}}]`
```
Comment (EmilySeville7cfg) [inline pages/linux/dm-tool.md:9]:
```suggestion
- Switch to the specified user, showing an authentication prompt if required:

`dm-tool switch-to-user {{username}} [{{session}}]`
```

File: pages/linux/dm-tool.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+- lock the current session
+`dm-tool lock`
```
Comment (EmilySeville7cfg) [inline pages/linux/dm-tool.md:12]:
```suggestion
- Lock the current session:

`dm-tool lock`
```

File: pages/linux/dm-tool.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+
+- add a new X server instance
+`dm-tool add-seat [{{name}}={{value}}]`
```
Comment (EmilySeville7cfg) [inline pages/linux/dm-tool.md:15]:
```suggestion
- Add a new X server instance:

`dm-tool add-seat [{{name}}={{value}}]`
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

**Comment by godalming123** [comment]
This is fixed in add-dm-tool branch

**Comment by godalming123** [comment]
the new PR with the actual branch is [here](https://github.com/tldr-pages/tldr/pull/7710)

**Pull request closed without merging**